devs/docs/es-backports.rst

```diff
@@ -190,25 +190,25 @@ should be crossed out as well.
 - [x] b21cb7f4666 Reduce allocations when persisting cluster state (#61159)
 - [x] c6276ae1771 Fail invalid incremental cluster state writes (#61030)
 - [ ] 25404cbe3d0 Provide option to allow writes when master is down (#60605)
-- [ ] 6644f2283d2 Do not access snapshot repo on dedicated voting-only master node (#61016)
+- [x] 6644f2283d2 Do not access snapshot repo on dedicated voting-only master node (#61016)
 - [ ] af519be9cbd Ensure repo not in use for wildcard repo deletes (#60947)
 - [ ] 32423a486d6 Simplify and Speed up some Compression Usage (#60953) (#61008)
-- [ ] 2fa6448a15b System index reads in separate threadpool (#60927)
-- [ ] a93be8d5771 Handle nested arrays in field retrieval. (#60981)
-- [ ] 3e2dfc6eac0 Remove GCS Bucket Exists Check (#60899) (#60914)
-- [ ] 0286d0a769e Move distance_feature query building into MFT (#60614) (#60846)
-- [ ] f44c28b5951 Deprecate and ignore join timeout (#60872)
+- [s] 2fa6448a15b System index reads in separate threadpool (#60927)
+- [s] a93be8d5771 Handle nested arrays in field retrieval. (#60981)
+- [s] 3e2dfc6eac0 Remove GCS Bucket Exists Check (#60899) (#60914)
+- [s] 0286d0a769e Move distance_feature query building into MFT (#60614) (#60846)
+- [s] f44c28b5951 Deprecate and ignore join timeout (#60872)
 - [x] 1f49e0b9d01 Fix testRerouteOccursOnDiskPassingHighWatermark (#60869)
 - [ ] 2f76c48ea71 Propagate forceExecution when acquiring permit (#60634)
```

Comment:
This and some other patches will be done separately as have more conflicts. I did some trivial patches in this PR

Reply:
We didn't backport the indexing memory tracking changes which contain `IndexingPressure`, so I think you'll have to skip this patch for now

Reply:
> indexing memory tracking changes which contain IndexingPressure

 is it ^ related to Primary-replica resync patch?https://github.com/elastic/elasticsearch/commit/2f76c48ea71d5b45ef531ab4c579022e82757438?
 
I saw additions of this flag in https://github.com/crate/crate/pull/10356/commits/206ee29e167c3cb01695af77b1207d086177e871#diff-be14860e9cb55a6ae2b5745414f56f59065480b48027efa6d8ebb3061c00c4d7R143 and thought it's relevant for us.

Reply:
It's related to https://github.com/elastic/elasticsearch/commit/1ef2cd7f1ad which we skipped because we've our own TransportShardUpsertAction. 

WriteLimits got later changed to IndexingPressure in https://github.com/elastic/elasticsearch/commit/623df95a3231cff0e4d50542e16a355f6e2a6211

So far the plan was to skip this stuff, and once we're done with the backports (+ maybe our indexing changes) and then see what parts of it are useful without doing a 1:1 port.

Reply:
thanks for the clarification!